### ISSUE TITLE
`wwctl configure -a` and `wwctl configure tftp` not creating `/var/lib/tftpboot/warewulf`

### ISSUE THREAD

**Issue by pedroalvesbatista** [warewulf/warewulf#756]
### Version of Warewulf

What version of Warewulf are you using? Run

```
wwctl version
wwctl version:	 4.4.0-1.git_dd3b34a
rpc version: apiPrefix:"rc1" apiVersion:"1" warewulfVersion:"4.4.0-1.git_dd3b34a"
```

### Expected behavior

Running `wwctl configure -a` and `wwctl configure tftp` should create `/var/lib/tftpboot/warewulf`.

### Actual behavior

An error message appears after running `wwctl configure tftp` as after all configurations done, compute nodes doesn't boot due to not finding any PXE file to load in :  

```
Writing PXE files to: /var/lib/tftpboot/warewulf
ERROR  : Could not open source file /usr/local/share/x86_64.efi: open /usr/local/share/x86_64.efi: no such file or directory
WARN   : ipxe binary could not be copied, booting may not work: open /usr/local/share/x86_64.efi: no such file or directory
ERROR  : Could not open source file /usr/local/share/x86_64.kpxe: open /usr/local/share/x86_64.kpxe: no such file or directory
WARN   : ipxe binary could not be copied, booting may not work: open /usr/local/share/x86_64.kpxe: no such file or directory
ERROR  : Could not open source file /usr/local/share/arm64.efi: open /usr/local/share/arm64.efi: no such file or directory
WARN   : ipxe binary could not be copied, booting may not work: open /usr/local/share/arm64.efi: no such file or directory
Enabling and restarting the TFTP services
```

### Steps to reproduce this behavior

* Run `wwctl configure -a` and `wwctl configure tftp`

### What OS/distro are you running

```sh
$ cat /etc/os-release
NAME="Rocky Linux"
VERSION="8.6 (Green Obsidian)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="8.6"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Rocky Linux 8.6 (Green Obsidian)"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:rocky:rocky:8:GA"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
ROCKY_SUPPORT_PRODUCT="Rocky Linux"
ROCKY_SUPPORT_PRODUCT_VERSION="8"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8"

```

### How did you install Warewulf

`dnf install https://github.com/hpcng/warewulf/releases/download/v4.4.0/warewulf-4.4.0-1.git_dd3b34a.el8.x86_64.rpm
`

**Comment by anderbubble** [comment]
Not verified yet, but marking as a bug for now.

**Comment by mslacken** [comment]
This is not error for creating the directory bu copying the iPXE binaries to it.
This issue is addressed by #931

**Comment by JasonYangShadow** [comment]
it looks like this issue gets fixed, I tested with nightly build
```
[vagrant@localhost ~]$ sudo dnf install -y warewulf-4.5.x-1.git_7d17bcc.el9.x86_64.rpm
Last metadata expiration check: 0:04:38 ago on Wed 24 Jan 2024 08:51:25 AM UTC.
Dependencies resolved.
================================================================================================================================================================================================== Package                                            Architecture                        Version                                                   Repository                                 Size
==================================================================================================================================================================================================Installing:
 warewulf                                           x86_64                              4.5.x-1.git_7d17bcc.el9                                   @commandline                               21 M
Installing dependencies:
 ipxe-bootimgs-aarch64                              noarch                              20200823-9.git4bd064de.el9                                appstream                                 124 k

Transaction Summary
==================================================================================================================================================================================================Install  2 Packages

Total size: 21 M
Total download size: 124 k
Installed size: 71 M
Downloading Packages:
ipxe-bootimgs-aarch64-20200823-9.git4bd064de.el9.noarch.rpm                                                                                                       149 kB/s | 124 kB     00:00
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------Total                                                                                                                                                              94 kB/s | 124 kB     00:01
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                                                                          1/1
  Installing       : ipxe-bootimgs-aarch64-20200823-9.git4bd064de.el9.noarch                                                                                                                  1/2
  Running scriptlet: warewulf-4.5.x-1.git_7d17bcc.el9.x86_64                                                                                                                                  2/2
  Installing       : warewulf-4.5.x-1.git_7d17bcc.el9.x86_64                                                                                                                                  2/2
  Running scriptlet: warewulf-4.5.x-1.git_7d17bcc.el9.x86_64                                                                                                                                  2/2
  Verifying        : ipxe-bootimgs-aarch64-20200823-9.git4bd064de.el9.noarch                                                                                                                  1/2
  Verifying        : warewulf-4.5.x-1.git_7d17bcc.el9.x86_64                                                                                                                                  2/2

Installed:
  ipxe-bootimgs-aarch64-20200823-9.git4bd064de.el9.noarch                                                 warewulf-4.5.x-1.git_7d17bcc.el9.x86_64

Complete!
[vagrant@localhost ~]$ whereis wwctl
wwctl: /usr/bin/wwctl /usr/local/bin/wwctl /usr/share/man/man1/wwctl.1.gz
[vagrant@localhost ~]$ sudo wwctl configure tftp
Writing PXE files to: /var/lib/tftpboot/warewulf
Enabling and restarting the TFTP services
```

RPM build here
`https://github.com/hpcng/warewulf/releases/download/nightly/warewulf-4.5.x-1.git_7d17bcc.el9.x86_64.rpm`

OS version:
```
[vagrant@localhost ~]$ cat /etc/os-release
NAME="Rocky Linux"
VERSION="9.3 (Blue Onyx)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="9.3"
PLATFORM_ID="platform:el9"
PRETTY_NAME="Rocky Linux 9.3 (Blue Onyx)"
ANSI_COLOR="0;32"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:rocky:rocky:9::baseos"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
SUPPORT_END="2032-05-31"
ROCKY_SUPPORT_PRODUCT="Rocky-Linux-9"
ROCKY_SUPPORT_PRODUCT_VERSION="9.3"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="9.3"
```

**Comment by franciscoadasme** [comment]
Hey there, I confirm that this issue is resolved as of commit adad227. I stumbled at this very problem with a version downloaded last Friday (Jan 26th). Using the newest version as of today solved the issue.

**Comment by mslacken** [comment]
Thanks for the follow up.
